src/controllers/v1/entities.js

```diff
@@ -856,7 +857,8 @@ module.exports = class Entities extends Abstract {
 					req.query.type ? req.query.type : '',
 					req.searchText,
 					req.pageSize,
-					req.pageNo
+					req.pageNo,
+					req.query.language
```

Comment:
@MallanagoudaB handle language missing situation just like how "type" is handled here